dashboard/src/scenes/person/list.js

```diff
@@ -293,11 +290,30 @@ const List = () => {
             sortOrder,
             sortBy,
             render: (p) => {
-              if (p.outOfActiveList)
-                return <div style={{ color: theme.black50 }}>{formatDateWithFullMonth(p.followedSince || p.createdAt || '')}</div>;
+              if (p.outOfActiveList) return <div className="tw-text-black50">{formatDateWithFullMonth(p.followedSince || p.createdAt || '')}</div>;
               return formatDateWithFullMonth(p.followedSince || p.createdAt || '');
             },
           },
+          {
+            title: 'Dernière interaction',
+            dataKey: 'lastUpdateCheckForGDPR',
+            onSortOrder: setSortOrder,
+            onSortBy: setSortBy,
+            sortOrder,
+            sortBy,
+            render: (p) => {
+              return (
+                <div
+                  className={
+                    dayjsInstance(p.lastUpdateCheckForGDPR).isAfter(dayjsInstance().add(-2, 'year'))
+                      ? 'tw-font-bold tw-text-red-500'
+                      : 'tw-text-black50'
```

Comment:
Je crois que c'est l'inverse qu'on veut !

```suggestion
                    dayjsInstance(p.lastUpdateCheckForGDPR).isAfter(dayjsInstance().add(-2, 'year'))
                      ? 'tw-text-black50'
                      : 'tw-font-bold tw-text-red-500'
```

Reply:
bien vu !